### PR TITLE
Switch to OpenTelemetry as upstream dropped OpenTracing support in GraphQL

### PR DESCRIPTION
**Pull request by rsvoboda**
Switch to OpenTelemetry as upstream dropped OpenTracing support in GraphQL

Fixes daily runs.

### CODE REVIEW

File: jdk17/graphql/src/test/java/io/quarkus/ts/http/graphql/TracingIT.java [hunk 1]
```diff
@@ -39,18 +38,23 @@ void verifyTracesInJaegerTest() {
...
+                    .queryParam("limit", 10)
+                    .queryParam("service", "graphql")
+                    .get(jaeger.getTraceUrl());
+            JsonPath body = traces.body().jsonPath();
+            Assertions.assertEquals(2, body.getList("data").size());
+            Assertions.assertEquals(2, body.getList("data[0].spans").size());
+            Assertions.assertEquals(2, body.getList("data[1].spans").size());
+            // Span with "operationName": "GraphQL" is child of the span with "operationName": "POST /graphql"
+            Assertions.assertTrue(body.getString("data[0].spans[0].operationName").equals(operation) ||
+                    body.getString("data[0].spans[0].operationName").equals("GraphQL"));
```
Comment (fedinskiy) [inline jdk17/graphql/src/test/java/io/quarkus/ts/http/graphql/TracingIT.java:55]: We do not check for `friend` and `friend_r`. Is that intentional?

File: jdk17/graphql/src/test/java/io/quarkus/ts/http/graphql/TracingIT.java [hunk 2]
```diff
@@ -39,18 +38,23 @@ void verifyTracesInJaegerTest() {
         Assertions.assertEquals("Plato", classic.jsonPath().getString("data.friend.name"));
         Assertions.assertEquals("Plato", reactive.jsonPath().getString("data.friend_r.name"));
 
-        // the tracer inside the application doesn't send traces to the Jaeger server immediately,
-        // they are batched, so we need to wait a bit
-        await().atMost(60, TimeUnit.SECONDS).untilAsserted(() -> {
-            Response response = when()
-                    .get(jaeger.getTraceUrl() + "?service=" + SERVICE_NAME);
-            Assertions.assertEquals(HttpURLConnection.HTTP_OK, response.statusCode());
```
Comment (fedinskiy) [inline jdk17/graphql/src/test/java/io/quarkus/ts/http/graphql/TracingIT.java:47]: Status code check is lost during the changes

File: jdk17/graphql/src/test/java/io/quarkus/ts/http/graphql/TracingIT.java [hunk 3]
```diff
@@ -39,18 +38,23 @@ void verifyTracesInJaegerTest() {
         Assertions.assertEquals("Plato", classic.jsonPath().getString("data.friend.name"));
         Assertions.assertEquals("Plato", reactive.jsonPath().getString("data.friend_r.name"));
 
-        // the tracer inside the application doesn't send traces to the Jaeger server immediately,
-        // they are batched, so we need to wait a bit
-        await().atMost(60, TimeUnit.SECONDS).untilAsserted(() -> {
-            Response response = when()
-                    .get(jaeger.getTraceUrl() + "?service=" + SERVICE_NAME);
```
Comment (fedinskiy) [inline jdk17/graphql/src/test/java/io/quarkus/ts/http/graphql/TracingIT.java:46]: It loos like `SERVICE_NAME` is not used anywhere anymore. Shouldn't we drop the constant?

**Review by fedinskiy — CHANGES_REQUESTED** [review]

**Comment by rsvoboda** [comment]
I will adjust the PR based on the feedback, I wanted to allign it as much as possible with the scenario from our main TS

File: jdk17/graphql/src/test/java/io/quarkus/ts/http/graphql/TracingIT.java (hunk 2 above)
Comment (rsvoboda) [reply jdk17/graphql/src/test/java/io/quarkus/ts/http/graphql/TracingIT.java:47]: will add

File: jdk17/graphql/src/test/java/io/quarkus/ts/http/graphql/TracingIT.java (hunk 3 above)
Comment (rsvoboda) [reply jdk17/graphql/src/test/java/io/quarkus/ts/http/graphql/TracingIT.java:46]: `quarkus.jaeger.service-name=graphql-service` was dropped, I will remove the constant

**Review by fedinskiy — APPROVED** [review]

**Pull request merged**